docs/04-upgrading/upgrading_to_v2.md

```diff
@@ -28,6 +31,7 @@ Attributes suffixed with `_millis` were renamed to remove said suffix and have t
 - `Actor.start`, `Actor.call`, `Actor.start_task`, `Actor.set_status_message` and `Actor.abort` return instances of the `ActorRun` model instead of an untyped `dict`.
 - Upon entering the context manager (`async with Actor`), the `Actor` puts the default logging configuration in place. This can be disabled using the `configure_logging` parameter.
 - The `config` parameter of `Actor` has been renamed to `configuration`.
+- Event handlers registered via `Actor.on` will now receive Pydantic objects instead of untyped dicts. For example, where you would do `event['isMigrating']`, you should now use `event.is_migrating`
```

Comment:
This is intentionally vague - maybe we should expose the event models somehow so that we can link them from here. Currently, they are internal members of Crawlee.